code/modules/economy/requisition/rc_aid.dm

```diff
@@ -1,4 +1,11 @@
 ABSTRACT_TYPE(/datum/req_contract/aid)
+/**
+ * Aid contracts are one of the three standard contract types.
```

Comment:
Referring to this as 1/3 doesn't add much value and requires more maintenance if an additional type is added in the future.

Reply:
Rectified this and other instances.